docs/GOVERNANCE.md

```diff
@@ -159,51 +165,79 @@ The specifications produced will adhere to the following:
+  true for updates to existing specifications and for new specifications.
+
+- When review/testing is completed, create a PR (for the "main" branch") that:
+  - Modifies the repo's files to use the new version string (w/o `-rc#`) as
+    appropriate.
+  - Update [RELEASES.md](RELEASES.md) to mention the new release, and
+    reference the yet-to-be-created release tag.
+  - Update the appropriate `*/RELEASE_NOTES.md` file with the changes
+    for the release. The list can be generated via:
+    `git log --pretty=format:%s main...v0.1 | grep -v "Merge pull"`
+    by replacing "v0.1" with the name of the previous release. Or, use github's
+    [new release](https://github.com/cloudevents/spec/releases/new) process
+    to generate the list w/o actually creating the release yet.
+
 - Merge the PR.
-- Create a [new release](https://github.com/cloudevents/spec/releases/new):
-  - Choose a "Tag version" of the form: `<subject>/vX.Y.Z`, e.g.
-    `cloudevents/v1.0.4` or `subscriptions/v1.0.0`
-  - Target should be `main`, the default value
-  - Release title should be the same as the Tag - `<subject>/vX.Y.Z`
-  - Add some descriptive text, or the list of PRs that have been merged since
-    the previous release. The git query to get the list commits since the last
-    release is:
-    `git log --pretty=format:%s main...v0.1 | grep -v "Merge pull"`.
-    Just replace "v0.1" with the name of the previous release.
-  - Press `Publish release` button
-- Create an "announcement" highlighting the key features of the new release and
-  any potential noteworthy activities of the group:
-  - Send it to the mailing list
-  - Announce the release on our
-    [twitter account](http://twitter.com/cloudeventsio)
+  - Note that the link checker should fail since any references to the new
+    release tag will not be valid yet. This is expected.
+
+- Create a new branch with the same name as the new release version string
+  appended with `-branch` (e.g. `<subject>/vX.Y.Z-branch`).
+  - Use Github to create a
+    [new release](https://github.com/cloudevents/spec/releases/new).
+    During that process, create a new tag with the new release version
+    string (e.g. `<subject>/vX.Y.Z`) w/o any suffix.
+  - Rerun the github CI actions from the previous PR and the "main" branch as
+    they should all pass now; as a sanity check.
+
+- Create an "announcement" highlighting the key features of the new release
+  and any potential noteworthy activities of the group:
+  - Send it to the mailing lists.
+  - Announce the release on our [X account](http://x.com/cloudeventsio).
   - Add it to the "announcement" section of our
-    [website](https://cloudevents.io/)
+    [website](https://cloudevents.io/).
+
+- If an update to a release is needed, create a PR for the appropriate
+  branches (including "main"), and merge when ready. For any release that's
+  updated, you'll need to move the tag for that release to point to the HEAD
```

Comment:
I *think* when referring to it normally, this should be head (or tip) rather than HEAD. I think HEAD is just the committish you can use in git commands.

```suggestion
  updated, you'll need to move the tag for that release to point to the head
```

Reply:
done